cmd/kubeadm/app/preflight/checks.go

```diff
@@ -291,7 +291,7 @@ func (fac FileAvailableCheck) Check() (warnings, errors []error) {
 	glog.V(1).Infof("validating the existence of file %s", fac.Path)
 	errors = []error{}
 	if _, err := os.Stat(fac.Path); err == nil {
-		errors = append(errors, fmt.Errorf("%s already exists", fac.Path))
+		errors = append(errors, fmt.Errorf("%s already exists, kubeadm init and join should not be called on the same machine", fac.Path))
```

Comment:
This is misleading. I mean this error may not caused by using `kubeadm init` and `kubeadm join` together.

What if we are running `kubeadm init` twice?

We'd better give some hints to the users.
@kubernetes/sig-cluster-lifecycle-pr-reviews How about changing to below,

```
fmt.Errorf("%s already exists. Be aware to issue 'kubeadm reset' to clean such legacy files.", fac.Path)
```

Reply:
You are right. thanks!